lib/openstudio/workflow/version.rb

```diff
@@ -35,6 +37,6 @@
 
 module OpenStudio
   module Workflow
-    VERSION = '1.3.5'.freeze # Suffixes must have periods (not dashes)
+    VERSION = '2.0.0-pre1' # Suffixes must have periods (not dashes)
```

Comment:
This seems to be contradicting the comment

Reply:
ha! valid point. i'll fix this.